nixos/modules/services/networking/webhook.nix

```diff
@@ -0,0 +1,159 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.webhook;
+  defaultUser = "webhook";
+
+  hooksFile = pkgs.writeText "webhook-config" cfg.hooks;
+
+  # https://github.com/adnanh/webhook/blob/master/docs/Hook-Rules.md
+  # We might merge with "or" (perhaps?), but until then, allow only a single definition.
+  typeTriggerRule = types.uniq types.attrs;
+
+in {
+  options = {
+    services.webhook = {
+      enable = mkEnableOption (mdDoc ''
+        [Webhook] is server written in Go, that allows you to create HTTP endpoints (hooks),
+        which execute configured commands for any person or service that knows the URL.
+
+        [Webhook]: https://github.com/adnanh/webhook
+      '');
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.webhook;
+        defaultText = "pkgs.webhook";
+        example = literalExample "pkgs.webhook";
```

Comment:
```suggestion
        example = literalExpression "pkgs.webhook";
```

Reply:
Out of curiosity, what's the difference?

Reply:
```
trace: warning: literalExample is deprecated, use literalExpression instead, or use literalDocBook for a non-Nix description.
```

Reply:
Thanks. Updated